src/components/side_panel/table_panel/table_panel.xml

```diff
@@ -84,6 +85,7 @@
         />
       </Section>
       <div class="o-sidePanelButtons">
+        <button t-on-click="resetTable" class="o-table-reset o-button">Reset table</button>
```

Comment:
Not sure we really want a "reset table" button ? It doesn't seems very useeful, does not functionnaly make sense that it would only reset the range and not all the other options, and there should aready be a reset option on the selection input.

Also you should add a test that would address the actual crash.